docs/utilities/tags.md

```diff
@@ -0,0 +1,53 @@
+Tags
+====
+
+Tags are generalized sets of objects in the game, used for grouping related things together and providing fast membership checks.
+
+Declaring Your Own Groupings
+----------------------------
+Tags are declared in your mod's datapack. For example, `/data/modid/tags/blocks/foo/tagname.json` will declare a `Tag<Block>` with ID `modid:foo/tagname`.
+Similarly, you may append to or override tags declared in other domains, such as vanilla, by declaring your own JSONs.
```

Comment:
vanilla -> Vanilla? I think it's capitalized almost everywhere, so better keep it consistent.
Also, I'd go with "writing your own JSONs" or something, you don't just declare the JSON file.